lib/node_modules/@stdlib/utils/parse-ndjson/docs/repl.txt

```diff
@@ -0,0 +1,57 @@
+{{alias}}( str[, reviver] )
+    Attempts to parse a string as Newline-Delimited JSON (NDJSON).
+
+    Function behavior differs from `JSON.parse()` as follows:
+
+    - Returns array of parsed JSON
+    - Throws a `TypeError` if provided any value which is not a string.
+    - Returns, rather than throws, a `SyntaxError` if unable to parse a string as NDJSON.
+    - Throws a `TypeError` if provided a `reviver` argument which is not a function.
+
+    Parameters
+    ----------
+    `str`: `string`
+        String to parse as Newline-Delimited JSON.
+
+    reviver: Function (optional)
+        Transformation function.
+
+    Returns
+    -------
+    `out`: `Array<*>|Error`
+        Array of Parsed values or an error.
+
+    Examples
+    --------
+
+    Example 1: Parsing Valid NDJSON
```

Comment:
See the REPL text guide. Use ES5. Heading such as this must be valid JavaScript comments.

Reply:
Done